src/matrixvariate.jl

```diff
@@ -136,8 +140,8 @@ end
 
 ## Custom adjoint since Zygote can't differentiate through `@warn`
 # TODO: Remove when fixed upstream in Distributions
-ZygoteRules.@adjoint function Wishart(df::T, S::AbstractPDMat{T}, warn::Bool = true) where T
-    function _Wishart(df::T, S::AbstractPDMat{T}, warn::Bool = true) where T
+ZygoteRules.@adjoint function Wishart(df::T, S::AbstractPDMat{T}, warn::Bool = true) where {T <: Real}
+    function _Wishart(df::T, S::AbstractPDMat{T}, warn::Bool = true)
```

Comment:
Revert this change and add the inner `where` clause again? The function `_Wishart` was intended to be defined for all types `T`, possibly different from the outer type parameter `T`.